src/protocol/protocol.yml

```diff
@@ -242,7 +242,9 @@ FetchRequest:
           type: array?
           items: NameValue
         postData: binary?
+        jsonData: json?
         formData: json?
```

Comment:
Why don't we make `formData` and `multipartData` strongly-typed? This will produce nice errors when user passes some `form: { foo: Buffer, bar: { x: 3 }, baz: childProcess }`.

Reply:
Done.